Krivorotova Tatyana/largest.js

```diff
@@ -0,0 +1,17 @@
+/*     largest     */
+function largest(arr)
+{
+    let numbers = [];
+    for(let i = 0; i < arguments.length; ++i)
```

Comment:
what's  the propose of transforming `arguments` to real array? you didn't use any of array methods.